app/styles/components/list.css

```
@@ -1,10 +1,12 @@
 .list {
-  --list-spacing:  0.25em;
+  --list-spacing: 0.25em;
 
   margin: 0;
   list-style-type: none;
   font-size: inherit;
   padding-inline-start: var(--spacing-sm);
+  line-height: var(--line-height-normal);
+  font-size: var(--font-size-base);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)